src/hmac/rtl/hmac_reg.rdl

```diff
@@ -64,54 +62,45 @@ addrmap hmac_reg {
 
     /* ---- HMAC384 Component Control ---- */
     reg {
-        name = "HMAC384 component control register type definition";
-        desc = "One 3-bit register including the following flags: 
-                bit #0: INIT : Trigs the HMAC384 core to start the 
-                                processing for the key and the first padded 
-                                message block.
-                bit #1: NEXT: ​Trigs the HMAC384 core to start the 
-                                processing for the remining padded message block. 
-                bit #3: Zeroize all internal registers after HMAC process, to avoid SCA leakage.
-                This register is located at HMAC384_base_address + 0x0000_0010
-                After each software write, hardware will erase the register.";
+        name = "HMAC384 CTRL";
+        desc = "HMAC384 component control register type definition";
 
         default sw = w;
         default hw = r;
         default resetsignal = reset_b;
-        field {desc = "Control init command bit"; singlepulse;} INIT = 1'b0;
-        field {desc = "Control next command bit"; singlepulse;} NEXT = 1'b0;
-        field {desc = "Zeroize all internal registers"; singlepulse;} ZEROIZE = 1'b0;
+        field {desc = "Control init command bit: Trigs the HMAC384 core to start the 
+                                processing for the key and the first padded 
+                                message block."; singlepulse;} INIT = 1'b0;
+        field {desc = "Control next command bit: ​Trigs the HMAC384 core to start the 
+                                processing for the remining padded message block. "; singlepulse;} NEXT = 1'b0;
+        field {desc = "Zeroize all internal registers: Zeroize all internal registers after HMAC process, to avoid SCA leakage."; singlepulse;} ZEROIZE = 1'b0;
```

Comment:
Add this back? "After each software write, hardware will erase the register."

Reply:
singlepulse definition is added instead of this.